index.js

```diff
@@ -0,0 +1,130 @@
+const inputBox = document.querySelector(".input-field input");
+const addButton = document.querySelector(".input-field button");
+const deleteAllButton = document.querySelector(".footer button");
+const toDoList = document.querySelector(".to-do-list");
+const completedTaskList = document.querySelector(".completed-task-list");
+const pendingTask = document.querySelector(".footer span");
+const saveButton = document.querySelector(".save-button");
+var taskListArray;
+var completeTaskList;
+var updateindex;
+
+if (localStorage.getItem("Task list")) {
+    taskListArray = JSON.parse(localStorage.getItem("Task list"));
+}
+else {
+    taskListArray = JSON.parse("[]");
+}
+
+if (localStorage.getItem("Completed task")) {
+    completeTaskList = JSON.parse(localStorage.getItem("Completed task"));
+}
+else {
+    completeTaskList = JSON.parse("[]");
+}
+
+
+let userEnteredTask;
+inputBox.onkeyup = () => {
```

Comment:
Please define this function separately and then assign it to onKeyUp property.
As, by function name then it will help in understanding that what is happening while "onKeyUp"